source/agora/test/Base.d

```diff
@@ -564,6 +567,25 @@ public class TestAPIManager
                     idx, node.getBlockHeight(), height), file, line));
     }
 
+    /// Ditto
+    public void expectBlock (Height height, const(BlockHeader) enrolled_header,
+        Duration timeout, string file = __FILE__, int line = __LINE__)
+    {
+        this.expectBlock(this.clients, height, enrolled_header, timeout, file,
+            line);
+    }
+
+    /// Ditto
+    public void expectBlock (Clients)(Clients clients, Height height,
+        const(BlockHeader) enrolled_header, Duration timeout,
+        string file = __FILE__, int line = __LINE__) if (isInputRange!Clients)
+    {
+        auto distance = cast(ushort)(height - enrolled_header.height - 1);
```

Comment:
Could you add `assert(height > enrolled_header.height)`? Otherwise this will overflow and turn into `ulong.max`.

Reply:
Yes. I will. I think that's needed.